modules/programs/git.nix

```diff
@@ -290,6 +290,8 @@ in {
                   "ssl")
               else
                 "";
+              smtpSslCertPath =
+                if smtp.tls.enable then smtp.tls.certificatesFile else "";
```

Comment:
According to `git-config(1)`:
```
sendemail.smtpsslcertpath
          Path to ca-certificates (either a directory or a single file). Set it to
          an empty string to disable certificate verification.
```

Reply:
Ah an excellent point. Would `mkIf` work here to achieve what I want?

---

As an aside, if `tls` is enabled, it defaults to [`/etc/ssl/certs/ca-certificate.crt`](https://github.com/nix-community/home-manager/blob/master/modules/accounts/email.nix#L355), and if it is disabled, there is probably no certification verification to disable :wink:

I do agree it doesn't make sense to outright disable tls verification though.

Reply:
`mkIf` should work since `iniContent` is not a `types.attrs` but a structured type that handles `mkIf` and such.

Reply:
Awesome, this is now `mkIf` :sparkles: